core/src/builder.ts

```diff
@@ -77,13 +86,27 @@ function PropertyBuilder<Model>(initialProps?: Partial<Model>) {
 function Builder<Model>({
   generator,
   transformers,
+  type,
```

Comment:
In the new pattern, we will be receiving which representation this builder is for: rest | graphql